scripts/build/replaceCopyright.js

```diff
@@ -5,7 +5,7 @@ const yamlPath = "ui5.yaml";
 
 function getDefaultCopyRight() {
     const content = replaceCurrentYear(fs.readFileSync(yamlPath, 'utf8'));
-    const yamlContent = yaml.safeLoad(content);
+    const yamlContent = yaml.load(content);
```

Comment:
does the safeLoad no longer exists?
Does this belongs to this fix? If not I would create another PR fixing the load function first and then merge this PR

Reply:
@flovogt : i saw the error when i tried to run in local. anyways raised another pr #281